client/landing/jetpack-cloud/components/threat-item-header/index.tsx

```diff
@@ -8,49 +8,79 @@ import { translate } from 'i18n-calypso';
  * Internal dependencies
  */
 import { Threat } from 'landing/jetpack-cloud/components/threat-item/types';
-import { getThreatType } from 'landing/jetpack-cloud/components/threat-item/threat';
+import { getThreatType } from 'landing/jetpack-cloud/components/threat-item/utils';
 
 interface Props {
 	threat: Threat;
+	// Set isStyled to false if you care about strings only
+	isStyled: boolean;
 }
 
 // This should be temporary since this data should be coming from the api
 // and not something that we should change to accommodate the results.
-const ThreatItemHeader = ( { threat }: Props ): i18nCalypso.TranslateResult => {
+const ThreatItemHeader = ( { threat, isStyled = true }: Props ) => {
 	const { filename, extension = { slug: 'unknown', version: 'n/a' } } = threat;
 
 	const basename = filename ? filename.replace( /.*\//, '' ) : '';
 
 	switch ( getThreatType( threat ) ) {
 		case 'core':
-			return translate( 'Infected core file: {{filename/}} ', {
-				components: {
-					filename: <code className="threat-item-header__alert-filename">{ basename }</code>,
-				},
-			} );
+			return isStyled
+				? translate( 'Infected core file: {{filename/}}', {
+						components: {
+							filename: <code className="threat-item-header__alert-filename">{ basename }</code>,
+						},
+				  } )
+				: translate( 'Infected core file: %s', {
+						args: [ basename ],
+				  } );
 
 		case 'file':
-			return translate( 'The file {{filename/}} contains a malicious code pattern.', {
-				components: {
-					filename: <code className="threat-item-header__alert-filename">{ basename }</code>,
-				},
-			} );
+			return isStyled
+				? translate( 'The file {{filename/}} contains a malicious code pattern', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'The file {{filename/}} contains a malicious code pattern.' )` *ES Score: 10*